Src/AutoFixture/Kernel/MultipleRelay.cs

```diff
@@ -26,7 +26,7 @@ public int Count
             get { return this.count; }
             set
             {
-                if (value <= 0)
+                if (value < 0)
                 {
                     throw new ArgumentOutOfRangeException("value", "Count cannot be zero or negative.");

```

Comment:
*Holy crap!* It's now [updated](https://github.com/AutoFixture/AutoFixture/pull/368/files#diff-cb78a5c3ba434ba944c25a25d7f6422bR31). – Thank you!